sandbox.ipynb

```diff
@@ -209,9 +189,39 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.errorbar(glc.x, glc.y, yerr=gerr, linestyle='None', marker='o', label='Transient')\n",
-    "plt.errorbar(slc.x, slc.y, yerr=serr, linestyle='None', marker='+', label='Variable')\n",
-    "plt.legend()"
+    "# import numpy as np\n",
```

Comment:
nit. (here and below) maybe you can throw commented code into another notebook or python file?

Reply:
Fixed!  I'm a code hoarder and have trouble letting go of old stuff.